winpr/libwinpr/sspi/Kerberos/krb5glue_mit.c

```diff
@@ -143,15 +145,28 @@ krb5_error_code krb5glue_get_init_creds(krb5_context ctx, krb5_principal princ,
 		if (krb_settings->renewLifeTime)
 			krb5_get_init_creds_opt_set_renew_life(gic_opt, krb_settings->renewLifeTime);
 		if (krb_settings->withPac)
-			krb5_get_init_creds_opt_set_pac_request(ctx, gic_opt, TRUE);
+			rv = krb5_get_init_creds_opt_set_pac_request(ctx, gic_opt, TRUE);
```

Comment:
is it me of the check of the return value is not done ?

Reply:
no, just me forgetting to add it.